HPCC-13006 DOCS:Standardize right-click

g-pan opened the pull request:
FIX HPCC-13006 DOCS:Standardize right-click
Throughout documentation right-click should
appear the same way in all docs.

Signed-off-by: G Panagiotatos g.pan3.14@gmail.com

@JamesDeFabia please review

Commit:
HPCC-13006 DOCS:Standardize right-click

FIX HPCC-13006 DOCS:Standardize right-click
Throughout documentation right-click should
appear the same way in all docs.

Signed-off-by: G Panagiotatos <g.pan3.14@gmail.com>

richardkchapman commented:
You may not care (and I do not care) but you signed this commit with your gmail address rather than the ln address you usually use.

JamesDeFabia commented on docs/UsingConfigManager/UsingConfigManager.xml:
```diff
@@ -1535,8 +1535,8 @@ sudo -u hpcc cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/envi
             <listitem>
               <para>On the <emphasis role="bold">Instances</emphasis> tab,
-              Rt-click on the table on the right hand side, choose <emphasis
-              role="bold">Add Instances…</emphasis></para>
```
Add Instances... is correct. Do not remove the ellipsis.

Commit:
HPCC-13006 DOCS:Standardize right-click

FIX HPCC-13006 DOCS:Standardize right-click
Throughout documentation right-click should
appear the same way in all docs.

Signed-off-by: G Panagiotatos <g.pan3.14@gmail.com>

JamesDeFabia commented:
Looks good to merge
